### PR TITLE
syntax: make: support ::= variable syntax

### PR DESCRIPTION
**Pull request by rohieb**
See discussion in #5464

### CODE REVIEW

File: runtime/syntax/make.vim [hunk 1]
```diff
@@ -2,8 +2,8 @@
 " Language:	Makefile
 " Maintainer:	Roland Hieber <rohieb+vim-iR0jGdkV@rohieb.name>
```
Comment (k-takata) [inline runtime/syntax/make.vim]: How about adding your GitHub account here so that one can easily contact you on GitHub?

Comment (rohieb) [reply runtime/syntax/make.vim]: Good idea!

**Comment by bogdan-barbu** [comment]
The first colon is also highlighted in this case, not just the variable's name.

**Comment by brammool** [comment]
I'll include it, thanks.

**Comment by Boobies** [comment]
I commented on this yesterday but my comment seems to have not made it, weirdly. The first colon gets highlighted as well, not just the variable's name.

**Comment by rohieb** [comment]
> The first colon gets highlighted as well, not just the variable's name.

Hm. I cannot reproduce that here…
```
:version
VIM - Vi IMproved 8.1 (2018 May 18, compiled Jun 15 2019 16:41:15)
Included patches: 1-875, 878, 884, 948, 1046, 1365-1368, 1382, 1401
Modified by team+vim@tracker.debian.org
Compiled by team+vim@tracker.debian.org
```
cfcb6ca0-3856-11ea-bd69-271d77a566ef

**Comment by Boobies** [comment]
Sorry, I had a typo in my manual "patch" :) Disregard my last comment; it works as expected.

**Pull request closed without merging**